app/pages/sections/additional-services/select/additional-service/controller.js

```diff
@@ -1,3 +1,22 @@
+import { getData } from 'buying-catalogue-library';
+import { getEndpoint } from '../../../../../endpoints';
+import { logger } from '../../../../../logger';
 import { getContext } from './contextCreator';
 
 export const getAdditionalServicePageContext = params => getContext(params);
+
+export const findAdditionalServices = async ({ addedCatalogueSolutions, accessToken }) => {
+  const endpoint = getEndpoint({ api: 'bapi', endpointLocator: 'getAdditionalServices', options: { addedCatalogueSolutions } });
+  const { additionalServices } = await getData({ endpoint, accessToken, logger });
+  logger.info(`Found ${additionalServices.length} additional service(s).`);
+
+  return additionalServices;
+};
+
+export const findAddedCatalogueSolutions = async ({ orderId, accessToken }) => {
+  const endpoint = getEndpoint({ api: 'ordapi', endpointLocator: 'getAddedCatalogueSolutions', options: { orderId } });
+  const { catalogueSolutions } = await getData({ endpoint, accessToken, logger });
+  logger.info(`Found ${catalogueSolutions.length} catalogue solution(s).`);
```

Comment:
I would update this log message to make it clear its the added catalogue solutions found for a particular order id.